src/main/java/org/broadinstitute/dsm/util/externalShipper/GBFRequestUtil.java

```diff
@@ -507,6 +516,22 @@ public static Map getParticipants(String realm) {
         return participantsESData;
     }
 
+    public static Map getParticipants(String realm, String ddpParticipantId) {
+        DDPInstance ddpInstance = DDPInstance.getDDPInstanceWithRole(realm, DBConstants.NEEDS_NAME_LABELS);
+        Map<String, Map<String, Object>> participantsESData = null;
+        String filter = "AND profile.guid = '";
+
+        if (StringUtils.isNotBlank(ddpInstance.getParticipantIndexES())) {
+            StringBuilder builder = new StringBuilder();
+            builder.append(filter);
+            builder.append(ddpParticipantId);
+            builder.append("'");
+            String queryFiler = builder.toString();
+            participantsESData = ElasticSearchUtil.getFilteredDDPParticipantsFromES(ddpInstance, queryFiler);
```

Comment:
just use it like ElasticSearchUtil.getFilteredDDPParticipantsFromES(ddpInstance, ElasticSearchUtil.BY_GUID + kitRequest.getParticipantId()). no need for StringBuilder